TTS/utils/vad.py

```diff
@@ -0,0 +1,142 @@
+# This code is adpated from: https://github.com/wiseman/py-webrtcvad/blob/master/example.py
```

Comment:
Tests ?

Reply:
It is called by  TTS/bin/remove_silence_using_vad.py so the test of this is not necessary